core/storage/user/gae_models_test.py

```diff
@@ -534,6 +534,19 @@ def test_get_multi_failure(self):
 
         self.assertEqual(retrieved_object, [None, None])
 
+    def test_export_data_trivial(self):
+        user_data = user_models.UserSkillMasteryModel.export_data('fake_user')
+        test_data = {}
+        self.assertEqual(user_data, test_data)
+
+    def test_export_data_nontrivial(self):
+        user_data = user_models.UserSkillMasteryModel.export_data(self.USER_ID)
+        test_data = {
+            '%s.%s' % (self.USER_ID, self.SKILL_ID_1): self.DEGREE_OF_MASTERY,
```

Comment:
Let's hardcode the keys here to prevent confusion. (I.e. just use 'skill_id_1' instead of '%s.%s' % ...)